easybuild/easyconfigs/b/Beast/Beast-1.10.5pre1-GCC-11.3.0.eb

```
@@ -0,0 +1,35 @@
+# Author: Pablo Escobar Lopez (1/1)
+# Biozentrum - University of Basel
+easyblock = 'Tarball'
+
+name = 'Beast'
+version = '1.10.5pre1'
+
+homepage = 'https://beast.community'
+description = """ BEAST is a cross-platform program for Bayesian MCMC analysis of molecular
+ sequences. It is entirely orientated towards rooted, time-measured phylogenies inferred using
+ strict or relaxed molecular clock models. It can be used as a method of reconstructing phylogenies
+ but is also a framework for testing evolutionary hypotheses without conditioning on a single
+ tree topology. BEAST uses MCMC to average over tree space, so that each tree is weighted
+ proportional to its posterior probability. """
+
+toolchain = {'name': 'GCC', 'version': '11.3.0'}
+
+source_urls = ['https://github.com/%(namelower)s-dev/%(namelower)s-mcmc/releases/download/v%(version)s/']
+sources = ['BEASTv%s.tgz' % version[:-1]]
+checksums = ['1005e863919abffb6a0adc0c944eb22dbfd66cca9c53da996b5c5e7596da55b6']
+
+dependencies = [
+    # this is not mandatory but beagle-lib is recommended by developers
+    # beagle-lib will also load the required java dependency
+    # if you remove this you should add the java dependency
+    ('beagle-lib', '3.1.2'),
```

I agree with the suggestion by @jfgrimm , please consider the following changes
```suggestion
    ('Java', '11', '', SYSTEM),
    ('beagle-lib', '3.1.2'),  # optional, but recommended
```